package.json

```diff
@@ -1,11 +1,28 @@
 {
-  "name": "web-components",
+  "name": "@uswds/web-components",
   "private": true,
-  "version": "0.0.0",
+  "version": "1.0.0-alpha",
   "type": "module",
+  "files": [
+    ".storybook",
+    "storybook",
+    "scripts",
+    "dist",
+    "src"
+  ],
+  "exports": {
+    ".": {
+      "import": "./dist/components/index.js",
+      "default": "./dist/components/index.js"
+    },
+    "./components/*": "./dist/components/*.js",
```

Comment:
**question:** Should we remove `components` to shorten the import?

```suggestion
    "./*": "./dist/components/*.js",
```

```diff
// Before this change user imports via:
- import "@uswds/web-components/components/usa-banner";
+ import "@uswds/web-components/usa-banner"; 
```

Reply:
This is a good suggestion, and I've been thinking about it for a bit. The main reasons I included it are clarity around intent and leaving room to have other named exports like, say, `/styles` or `/fonts`. Maybe a YAGNI thing? I don't have strong feelings about it, beyond noting that having a `/components` export seems to be a common practice among other component-based design systems.